bolt-sidecar/src/config/limits.rs

```diff
@@ -36,6 +39,15 @@ pub struct LimitsOpts {
         default_value_t = LimitsOpts::default().min_priority_fee
     )]
     pub min_priority_fee: u128,
+    /// The maximum size in MiB of the [crate::state::ExecutionState] LRU cache that holds account
+    /// states. Each [crate::primitives::AccountState] is 48 bytes, and its key is 20 bytes, so the
+    /// default value of 1024 KiB = 1 MiB can hold around 15k account states.
+    #[clap(
+        long,
+        env = "BOLT_SIDECAR_MAX_ACCOUNT_STATES_SIZE",
+        default_value_t = LimitsOpts::default().max_account_states_size,
+    )]
+    pub max_account_states_size: NonZero<usize>,
```

Comment:
Do we really need this? I know the default will be used by 99.9% of users of this binary.
This feels more like an internal implementation detail than something users might want to set, but curious wdyt

Reply:
It's not super necessary but I think it's cool to have the option to do it.